### ISSUE TITLE
Controls that disclose additional content need an `aria-expanded` attribute: remove `aria-pressed`

### ISSUE THREAD

**Issue by afercia** [WordPress/twentytwenty#68]
See https://github.com/WordPress/twentytwenty/issues/66

All controls that reveal additional content need to communicate users their purpose and state.

The proper attribute to use for this purpose is [aria-expanded](https://www.w3.org/TR/wai-aria-1.1/#aria-expanded). 

[aria-pressed](https://www.w3.org/TR/wai-aria-1.1/#aria-pressed) has a different purpose and needs to be removed in favor of `aria-expanded`.

**Comment by afercia** [comment]
Note: applies also to the buttons to show the sub-levels in the menu.

**Comment by afercia** [comment]
Just noticed: `aria-expanded="true"` should not be used on the element that expands / collapses, as it's not an interactive element. It won't be perceived at all by assistive technologies. Example:

```
<div class="search-modal cover-modal show-modal active" data-modal-target-string=".search-modal" aria-expanded="true">
```

(note: it's also invalid HTML, as the div is not interactive and doesn't have an ARIA role suited for aria-expanded)

**Comment by andersnoren** [comment]
@afercia I've updated #71 to not trigger the `aria-expanded` attribute on the toggle target.
